### DISCUSSION TITLE
Logo?

### DISCUSSION THREAD

**Discussion by giann** [vnmakarov/mir#347] [Ideas]
I'd be glad to make one. Would you be opened to it @vnmakarov ?
Is the name of the project related to the space station?

**Comment by vnmakarov** [comment]
Thank you very much for your proposal.

MIR has many meanings in Russian.  The most common ones are world, peace, society.  Before Russian language simplification after revolution of 1917, the different meanings had different spellings.

There was a famous USSR space station.  So I guess space station theme would be ok for the logo.

Only I'd like to warn you that I will use the logo if I like it.

**Comment by giann** [comment]
> Only I'd like to warn you that I will use the logo if I like it.

Obviously 👍
